clusters/base/kube-prometheus-stack.yaml

```
@@ -22,8 +22,36 @@ spec:
       sourceRef:
         kind: HelmRepository
         name: prometheus
+
   values:
+    alertmanager:
+      alertmanagerSpec:
+        nodeSelector:
+          cncf-project: wg-green-reviews
+          cncf-project-sub: internal
+
```

```suggestion

    grafana:
      nodeSelector:
        cncf-project: wg-green-reviews
        cncf-project-sub: internal

    kube-state-metrics:
      nodeSelector:
        cncf-project: wg-green-reviews
        cncf-project-sub: internal
```

These settings are also needed.